mrr/src/main/java/gov/cdc/dex/mrr/VocabClient.kt

```diff
@@ -38,4 +46,26 @@ class VocabClient {
        // println("VocabKey :${vocabKey}")
         return vocabKey
     }
+
+    fun  setValueSetConcepts(valuesetConcepts: List<ValueSetConcept>,  key: String) {
+
+        var jedis = RedisUtility().redisConnection()
```

Comment:
try the same approach - with RedisUtility().redisConnection().use { ... }

Reply:
done